sylvia/tests/ui/attributes/payload/invalid_usage.stderr

```diff
@@ -0,0 +1,8 @@
+error: Missing payload parameter.
```

Comment:
Actually there is a payload attribute, not parameter ^^
But yeah, it's not 100% obvious, but it's such a niche error, I believe, that I thought it would be sufficient.
I will add check that no `sv` attributes were used on the `self` and `ctx` parameters.